awswrangler/distributed/ray/modin/s3/_read_parquet.py

```diff
@@ -1,17 +1,48 @@
 """Modin on Ray S3 read parquet module (PRIVATE)."""
+import logging
 from typing import TYPE_CHECKING, Any, Dict, Iterator, List, Optional, Union
 
 import modin.pandas as pd
 import pyarrow as pa
+import pyarrow.parquet
 from ray.data import read_datasource
 
+from awswrangler.distributed.ray import ray_remote
 from awswrangler.distributed.ray.datasources import ArrowParquetDatasource
 from awswrangler.distributed.ray.modin._utils import _to_modin
+from awswrangler.s3._read_parquet import _pyarrow_parquet_file_wrapper
 
 if TYPE_CHECKING:
     from mypy_boto3_s3 import S3Client
 
 
+_logger: logging.Logger = logging.getLogger(__name__)
+
+
+@ray_remote()
```

Comment:
This one is a nitpick, but it looks like we have decided to do the `ray_remote` directly in [register](https://github.com/aws/aws-sdk-pandas/blob/release-3.0.0/awswrangler/distributed/ray/_register.py#L40) since it's imported there but here is fine too